ports/darwin/au_darwin.go

```diff
@@ -1,17 +1,18 @@
-// Copyright 2018 The ZikiChombo Authors. All rights reserved.  Use of this source
+// Copyright 2018 The ZikiChomgo Authors. All rights reserved.  Use of this source
```

Comment:
Where are all the Chomgo's coming from? It seems to be a common copy-paste issue? Where are you copying from, perhaps update that to the correct name? And then run `grep -R -i comgo` on the source tree to find other occurrences.

Reply:
chomgo's come from a typo in a license header file used by a program I use to add license headers :)   Funny how typos can result in puns.